meson.build

```diff
@@ -204,36 +204,30 @@ if get_option('enable-python')
   # Check python 2.7
   python2_dep = dependency('python-2.7', required: false)
   if python2_dep.found()
-    python2_incs = run_command(pg_pkgconfig, ['python-2.7', '--cflags']).stdout().strip().split()
+    python2_incs = []
+    python2_incs += run_command(pg_pkgconfig, ['python-2.7', '--cflags']).stdout().strip().split()
+    python2_incs += run_command('python2.7', ['-c', 'import site\nfor i in site.getsitepackages(): print("-I" + i + "/numpy/core/include")']).stdout().strip().split()
+    python2_incs += '-I' + run_command('python2.7', ['-m', 'site', '--user-site']).stdout().strip() + '/numpy/core/include'
```

Comment:
If numpy Linux package (non-PIP, but RPM/DEB) is installed, these will disrupt include paths.
If you can find numpy headers without adding operations, don't add them.

Reply:
> If numpy Linux package (non-PIP, but RPM/DEB) is installed, these will disrupt include paths.
> If you can find numpy headers without adding operations, don't add them.

@myungjoo 

IMHO, these are not related to include paths if you mean the compiler options provided with -I. These are just a logic for finding numpy. ```python2-dep``` and ```python3-dep``` are only used for building nnstreamer-python2.7 and 3.

```bash
[82/140] ccache cc -Iext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_python3@sha \
-Iext/nnstreamer/tensor_filter -I../ext/nnstreamer/tensor_filter -Igst/nnstreamer \
-I../gst/nnstreamer -I/usr/include/python3.5m \
-I/usr/include/x86_64-linux-gnu/python3.5m \
-I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include \
-I/usr/include/gstreamer-1.0 -I/usr/lib/x86_64-linux-gnu/gstreamer-1.0/include \
-I/usr/include/orc-0.4 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 \
-Wall -Winvalid-pch -Werror -std=gnu89 -g '-DVERSION="1.0.0"' \
-Wredundant-decls -Wwrite-strings -Wformat -Wformat-nonliteral -Wformat-security \
-Winit-self -Waddress -Wno-multichar -Wvla -Wpointer-arith -Wmissing-declarations \
-Wmissing-include-dirs -Wmissing-prototypes -Wnested-externs -Waggregate-return \
-Wold-style-definition -Wdeclaration-after-statement '-DNNSTREAMER_CONF_FILE="etc/nnstreamer.ini"' \
-DHAVE_ORC=1 -DENABLE_TENSORFLOW=1 -DENABLE_TENSORFLOW_LITE=1 -DENABLE_PYTORCH=1 \
-DENABLE_CAFFE2=1 -fPIC -pthread -MD -MQ \
'ext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_python3@sha/tensor_filter_python.c.o' \
-MF 'ext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_python3@sha/tensor_filter_python.c.o.d' \
-o 'ext/nnstreamer/tensor_filter/584bea7@@nnstreamer_filter_python3@sha/tensor_filter_python.c.o'\
 -c /home/wook/Work/prj_nnstreamer/nnstreamer/ext/nnstreamer/tensor_filter/tensor_filter_python.c
```

Reply:
Ah.. gotcha..